newton/_src/utils/__init__.py

```
@@ -21,6 +21,81 @@
 from .topology import topological_sort
 
 
+@wp.func
+def transform_inertia(t: wp.transform, I: wp.spatial_matrix):
```

Ruff (0.12.2)

Ambiguous variable name: `I`

(E741)